### ISSUE TITLE
Duplication error in Adguard list

### ISSUE THREAD

**Issue by BirdInFire** [nickspaargaren/no-google#188]
There is many duplicate on adguard list I will take jib mobile example (at the end of the list

you have 47 rules, but 45 of them are useless in adguard rules because they are already covered by those 2

||jibe.google.com^
||jibemobile.com^

Who tell 

jibe.google.com and every subdomain are blocked
jibemobile.com and every subdomain are blocked

I think it would be interesting to have a script that check if the main domain for example jibemobile.com is in the adguard list and so not add subdomain like : x.jibemobile.com because it increase the size of the list for nothing.

**Comment by nickspaargaren** [comment]
You're right, let's leave this open. `||` is already blocking all subdomains:

https://github.com/AdguardTeam/AdGuardHome/wiki/Hosts-Blocklists#special-characters
